tests/websockets/test_graphql_transport_ws.py

```diff
@@ -156,6 +161,49 @@ async def test_connection_init_timeout_cancellation(
         assert ws.closed
 
 
+async def test_close_twice(
+    mocker: MockerFixture, request, http_client_class: Type[HttpClient]
+):
+    if sys.version_info < (3, 8):
+        pytest.skip("Task name was introduced in 3.8 and we need it for this test")
+
+    if http_client_class == AioHttpClient:
+        pytest.skip(
+            "Closing a AIOHTTP WebSocket from a task currently doesnt work as expected"
```

Comment:
how come, what happens there?

Reply:
Not really sure, I copied that skip from other tests (https://github.com/strawberry-graphql/strawberry/blob/main/tests/websockets/test_graphql_transport_ws.py#L103) that also test the timeout assuming that my test would have the same issue

I'll try without it though

Reply:
I removed and it worked fine =P